MamboBryan/Randomy
# #166219895 Adding UI design elements

#### What does this PR do?
It adds UI elements like TextViews, SeekBar and buttons
#### Description of Task to be completed?
- Adds TextViews to guide user on what to do
- Adds SeekBars so the user can set the number's range
- It creates a button to generate a random number
#### How should this be manually tested?
- Clone the repo into your machine
- Open the app with android studio and wait to complete the build
- Run the app into your desired device
- After a successful build, the LandingActivity will appear on your device screen
- Press the start button to got to the RangeActivity
- Notice the added UI elements 
#### What are the relevant pivotal tracker stories?
https://www.pivotaltracker.com/story/show/166219895
#### Screenshots (if appropriate)
![RangeActivity](https://user-images.githubusercontent.com/40160345/58290623-1707e980-7dc3-11e9-9de1-5072010cc49c.PNG)
